client/src/js/services/PeriodService.js

```diff
@@ -0,0 +1,14 @@
+angular.module('bhima.services')
+.service('PeriodService', PeriodService);
+
+PeriodService.$inject = ['PrototypeApiService'];
+
+function PeriodService(PrototypeApiService) {
+  var service = this;
+
+  // inherit methods from the
+  angular.extend(service, PrototypeApiService);
```

Comment:
Na, i'll move this.  It will not be used anyway - not sure how it got into this branch.